Altis_Life.Altis/stringtable.xml

```diff
@@ -812,6 +812,17 @@
             <Portuguese>Ação Cancelada</Portuguese>
             <Polish>Anulowano akcję</Polish>
         </Key>
+        <Key ID="STR_NOTF_COMMANDVIEW">
```

Comment:
Could you copy what you have in English for all of the other languages? That way they are filled in and will work. They can be translated later.